release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+VERSION="$(date +%Y.%m.%d)"
+LASTEST_VERSION="$(git tag --list | tail -n 1)"
+
+if [ "${VERSION}" = "${LASTEST_VERSION}" ]; then
+    echo "Nothing to release"
+    exit 0
+fi
+
+# Make sure local fork is updated
+git fetch -p --all
+git checkout master
+git merge upstream/master
+
+# Bump version number
+echo ${VERSION} > VERSION
+
+# Generate the package
+make package-clean package
+
+# Sanity check Pulp Smash packages on both Python 2 and Python 3
+for python in python{2,3}; do
+    virtualenv -p ${python} pulpsmash-${python}
+    source pulpsmash-${python}/bin/activate
+    for dist in dist/*.{whl,tar.gz}; do
```

Comment:
How about changing this to `for dist in dist/*; do`? It'll be a little shorter, and more importantly, it'll match what the `publish` make target does.